content/graphing/infrastructure/process.md

```diff
@@ -92,6 +92,39 @@ In the [dd-agent.yaml][12] manifest used to create the daemonset, add the follow
 
 Refer to the standard [daemonset installation][13] and the [Docker Agent][11] information pages for further documentation.
 
+### Process Arguments Scrubbing
+
+In order to avoid that sensitive data is sent to Datadog servers and displayed on the Live Processes page, a Data Scrubber tool is shipped with the Process Agent. This feature is enabled by default and any process argument that matches one of the following words has its value hidden.
+
+```
+"password", "passwd", "mysql_pwd", "access_token", "auth_token", "api_key", "apikey", "secret", "credentials", "stripetoken"
+```
+
+The matching process is case insensitive and to expand this list, use the `custom_sensitive_words` field on the `datadog.yaml` file. The user can also insert wildcards to widen the matching scope of a word. However, the use of a purely wildcard `*` is not supported.
```

Comment:
Change to "Users can also use wildcards `*` to define their own matching scope. However, single wildcard `*` is not supported."

Reply:
Change to "The matching is case insensitive and user can define their own list, using `cutom_sensitive_words` field in `datadog.yaml` file under `process_config` section".